packages/firebase_crashlytics/firebase_crashlytics_platform_interface/pubspec.yaml

```diff
@@ -8,16 +8,16 @@ environment:
   flutter: ">=1.12.13+hotfix.5 <2.0.0"
 
 dependencies:
+  collection: ^1.14.3
+  firebase_core: ^0.7.0-nullsafety.0
```

Comment:
Can you sync with master and then make sure all the `firebase_core*` dependencies in all the Crashlytics packages are on the latest nullsafety versions